lib/containers/utils.pm

```diff
@@ -179,4 +179,20 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+=head2 can_build_sle_base
+
+C<can_build_sle_base> should be used to identify if sle base image runs against a
+system that it does not support registration and SUSEConnect.
+In this case the build of the container engine is not going to work as the base images
+lacks of the repos.
+
+The call should return false if you try to test sle base container on osd but host is not sle
+true otherwise.
+
+=cut
+sub can_build_sle_base {
+    my $has_sle_registration = script_run("test -e /etc/zypp/credentials.d/SCCcredentials");
+    return check_os_release('sles', 'ID') && $has_sle_registration;
```

Comment:
```suggestion
    return check_os_release('sles', 'ID') && script_run("test -e /etc/zypp/credentials.d/SCCcredentials");
```

Reply:
Why do you want this in one line? the variable is used for two reasons. to give also the explanation of why we use this `test` and make line shorter.

Reply:
Done

Reply:
Ok, it was for simplicity, but you are right the variable name is self-explanatory. Ok to keep it.

Reply:
thanks